dts/bindings/display/gooddisplay,ek79652.yaml

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2021, Linumiz
+# SPDX-License-Identifier: Apache-2.0
+
+description: EK79652 EPD display controller
+
+compatible: "gooddisplay,ek79652"
+
+include: spi-device.yaml
+
+properties:
+    height:
+      type: int
+      required: true
+      description: Height in pixel of the panel driven by the controller
+
+    width:
+      type: int
+      required: true
+      description: Width in pixel of the panel driven by the controller
+
+    reset-gpios:
+      type: phandle-array
+      required: true
+      description: RESET pin.
+
```

Comment:
Please fix the multiline string formatting. See https://github.com/zephyrproject-rtos/zephyr/pull/36189.

Reply:
@mbolivar-nordic  fixed!